### ISSUE TITLE
Started a Minecolonies Server but getting rebounding lag locally and externally even before Colony is built 

### ISSUE THREAD

**Issue by Pockyman1** [ldtteam/minecolonies#6846]
### Prerequisites 

- [x] I am running the latest **alpha** version of MineColonies and Structurize for my Minecraft version.
- [x] I checked the MineColonies/Structurize [wiki](https://wiki.minecolonies.ldtteam.com/) and made sure my issue is not covered there.
- [x] I made sure that **this issue is not a duplicate of any existing issue.**

### Context 

- Minecraft Version:1.16.5
- MineColonies Version: 0.14.60-alpha-universal
- Structurize Version:0.13.148-alpha-universal

## Problem description
I started a Minecolonies server by downloading the files available on curse forge after downloading the client side files on GDlauncher I removed 3 mods Horsestandstill, BYG, Bettercaves and its associated configs, then started  the server like normal with lvl seed that i found and liked which is (5280986485207250521) after a while i found a good plains area that i choose to Terraform and flatten out for the Colony although few moments later I started getting server lag and would rebound ever so often I've increased the server ram to 7 gbs to 10gbs and is run on a 32 gb PC, on GD Launcher I boasted ram usage to about 10 GBs or so I had a friend join me to start playing and we'd both be receiving the same rebound lag if we did stuff too quickly even before any colony stuff was made. Note I Run the server and Play on the Same PC while friend connecting to me from Central  USA while i'm in west coast. After paying attention to log for a while noticed there was a ton of Pathfinding debug issues that would spam the log for long while, while players were online on the server 
[debug-4.log.gz](https://github.com/ldtteam/minecolonies/files/6243232/debug-4.log.gz)
similar situation will occur on a fresh world after a while
[debug-1.log.gz](https://github.com/ldtteam/minecolonies/files/6243240/debug-1.log.gz)

### Notes or related things
 I had coppied over a World save from a previous version and applied it to the new one after removing said mods, although considered this to possibly be the issue so, I recreated the world as a new one with the same versions and would still get lag early on without any thing built yet besides digging out part of a hill to test
Here are some logs from when friend was playing with me 
[2021-03-31-5.log.gz](https://github.com/ldtteam/minecolonies/files/6243179/2021-03-31-5.log.gz)
here are logs after they friend hopped off for night and i increased Ram to 10gbs to see if that would resolve lag
[2021-03-31-7.log.gz](https://github.com/ldtteam/minecolonies/files/6243204/2021-03-31-7.log.gz)
here is a test as fresh world with same seed with some noticeable lag here and there 
[2021-04-01-1.log.gz](https://github.com/ldtteam/minecolonies/files/6243209/2021-04-01-1.log.gz)

---
## Viewers

* Add a thumbs-up to the issue to vote for it. This helps the request become more visible to the team and doesn't clutter the comments.
* Add a comment if you have any insights or background information that isn't already part of the conversation.

**Comment by Raycoms** [comment]
Even before the colony is built, seems like a non MineColonies problem for me? Any additional details proving it's our fault? Spark profile or similar?

**Comment by someaddons** [comment]
minecolonies does nothing without colonies, frequent lag is often just badly configured memory settings. Also fill in the template.

**Comment by Pockyman1** [comment]
> minecolonies does nothing without colonies, frequent lag is often just badly configured memory settings. Also fill in the template.

I did fill out the template but then when I click submit everything was removed except for the context everything underneath for Problem description and notes got deleted

**Comment by someaddons** [comment]
yea you had put it into comment arrows which made it not display, fixed it

**Comment by someaddons** [comment]
check this guide: https://gist.github.com/someaddons/ad1dcb8e0a69542a3ba8e48a5e72e53a and post results here

**Comment by Pockyman1** [comment]
> check this guide: https://gist.github.com/someaddons/ad1dcb8e0a69542a3ba8e48a5e72e53a and post results here

From what I looked through on the guide I am getting is tick lag and I would start Rubberbanding if too much activity was occurring which frequently occurred when more than 1 player was connected. and while keeping the tickmonitoring On. I would notice that avg tick would spike up from over 100% of avg tick to over 200% or 300% and would sometimes jump to 958.52 ms (1818.93% increase from avg tick and would fluctuate between those numbers even if I was to just stand still. 
Here are the Spark Results from what I went to test and record the lag that was occurring
[spark activity.txt](https://github.com/ldtteam/minecolonies/files/6243813/spark.activity.txt)
Only thing I really changed in Server files were settings, replaced some colonist names and increase Ram from 4Gbs to 7gbs or 10gbs and set the level seed in server properties
